engine/templates_tests/stepCondition.yaml

```diff
@@ -76,3 +76,31 @@ steps:
               then:
                   this: FOO
               message: changed state matched correctly
+    stepFour:
+        dependencies: [stepThree:ANY]
+        description: a check against a list of acceptable values
+        custom_states: [LIST_MATCH,LIST_NOMATCH]
+        action:
+            type: echo
+            configuration:
+                output:
+                    items: 'foo,bar'
+        conditions:
+            - type: check
+              if:
+                  - value: '{{.step.this.output.items}}'
```

Comment:
the condition is backwards, isnt it?

foo IS IN foo,bar

Reply:
Indeed, it doesn't match the code. Fixing.